www/posts/2020-05-25-v2-release-day.md

```diff
@@ -26,11 +20,11 @@ npm i snowpack@latest --save-dev
 
 "Rethink established best practices." That was the famous antagonistic tweet directed at the React team back when React & JSX were first announced in 2013.
 
-It had been meant as a snarky burn against the project, but the React team turned it into a rallying cry. Pete Hunt even named [an entire JSConf talk](https://www.youtube.com/watch?v=DgVS-zXgMTk) after it, explaining why "rethinking best practices" was exactly what the web needed at that moment. 7 years later, and Pete was clearly proven right.
+It had been meant as a snarky burn against the project, but the React team turned it into a rallying cry. Pete Hunt even named [an entire JSConf talk](https://www.youtube.com/watch?v=DgVS-zXgMTk) after it, explaining why "rethinking best practices" was exactly what the web needed at that moment. 7 years later, Pete was proven right.
 
-**Snowpack is a chance to rethink best practices again, this time around web tooling.** The web has evolved over the last decade, but our build tooling hasn't kept up. The best practices of the previous decade no longer fit the requirements of today.
+**Snowpack set out to challenge best practices again, this time around web tooling.** The web has evolved over the last decade, but our build tooling hasn't kept up. The assumptions and limitations of the previous decade no longer fit the requirements of today.
 
-**Snowpack is rethinking the biggest outdated assumption of all: the idea that web developers NEED bundlers.** Bundling continues to be great production optimization, dating all the way back to the early days of the web. It should continue to be available to those who want it. But... it was only in the last decade (with the rise of Browserify and later, Webpack) that bundling became a *required* part of your development workflow. 
+**Snowpack proves that web developers don't NEED a bundler to build complex web applications.** The idea that a bundler is *required* for any meaningful web development only goes back the last decade (with the rise of Browserify and later, Webpack). Snowpack is a return to the norm, where bundling is an optional production optimization available to those who want it
```

Comment:
I noticed the word "only" is used a lot in the post, might want to replace a few instances